### PR TITLE
OLS-155: Set project line length

### PR DESCRIPTION
**Pull request by onmete**
## Description

Implements project line length for 100.

See various threads about this topic
- https://github.com/openshift/lightspeed-service/pull/96
- https://redhat-internal.slack.com/archives/C068JAU4Y0P/p1704277149765139
- https://redhat-internal.slack.com/archives/C068JAU4Y0P/p1705091283613439

## Type of change

- [X] Refactor

## TODO
This is on hold until all PRs from today are merged. Once they are, I'll fix the rest of the errors.
- [ ] Merge all PRs
- [ ] Fix the rest of linting errors

### CODE REVIEW

**Comment by onmete** [comment]
This is on hold until all PRs from today are merged. Once they are, I'll fix the rest of the errors.

File: ols/utils/config.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# we are disabling ruff check for long lines because of the commented
+# code at the bottom
+# ruff: noqa: E501
```
Comment (onmete) [inline ols/utils/config.py:3]: @syedriko please don't forget to remove this if your PR is merged after this one.

cc @bparees

**Comment by tisnik** [comment]
Mostly it looks ok, but IIRC @bparees insists on not to use `noqa` tricks

**Comment by tisnik** [comment]
/lgtm

(but pls see my previous comment)

**Comment by bparees** [comment]
/approve

unfortunately needs a rebase

**Pull request merged**